docs/platforms/emr_serverless.md

```diff
@@ -204,6 +400,10 @@ parameters:
   # path where logs are stored or should be stored (optional)
   logs: s3://amzn-test-bucket/logs
 
+  # name of your athena connection (optional, defaults to "athena-default")
```

Comment:
if I am correct, this should default to whatever value set in `default_connections.athena` in pipeline.yml, which would then default to `athena-default`, is that right?